src/plugin/memory-webview-main.ts

```diff
@@ -84,9 +87,52 @@ export class MemoryWebview {
                 }
             })
         );
+
+        vscode.window.registerCustomEditorProvider(manifest.EDITOR_NAME, this);
     };
 
-    public async show(initialMemory?: Partial<DebugProtocol.ReadMemoryArguments>): Promise<void> {
+    public async saveCustomDocument(_document: vscode.CustomDocument, _cancellation: vscode.CancellationToken): Promise<void> {}
+    public async saveCustomDocumentAs(_document: vscode.CustomDocument, _destination: vscode.Uri, _cancellation: vscode.CancellationToken): Promise<void> {}
+    public async revertCustomDocument(_document: vscode.CustomDocument, _cancellation: vscode.CancellationToken): Promise<void> {}
+    public async backupCustomDocument(
+        _document: vscode.CustomDocument,
+        _context: vscode.CustomDocumentBackupContext,
+        _cancellation: vscode.CancellationToken
+    ): Promise<vscode.CustomDocumentBackup> {
```

Comment:
I believe that TypeScript should allow us to omit all of the arguments here and still consider the implementation correct. I'd prefer that as it would be less cluttered, but I can see an argument for retaining the data about what an implementation might look like.

Reply:
Done